EventFilter/SiPixelRawToDigi/src/PixelUnpackingRegions.cc

```diff
@@ -81,9 +80,8 @@ void PixelUnpackingRegions::initialize(const edm::EventSetup& es) {
     es.get<SiPixelFedCablingMapRcd>().get(cablingMap);
```

Comment:
(from the static analyzer)
shouldn't this be in esConsumes/token as well?

Reply:
Yes. I think this is the first time we've encountered an ESTransientHandle being used. Looking at the code, though, this should NOT be an ESTransientHandle since it is holding onto memory from the data product.